src/zapatos/zapatosTest.ts

```diff
@@ -0,0 +1,6 @@
+import * as db from 'zapatos/db';
+import type * as s from 'zapatos/schema';
+import pool from 'pg-pool';
+
+const authors = db.sql<s.document.SQL, s.document.Selectable[]>`
+  SELECT * FROM ${"document"}`.run(pool);
```

Comment:
and move this into a `db.test.ts` file. I want the test to look like this:

```ts
import {db} from './db';

expect("add users", ()=>{
  const [user] = await db.insert('user', [
    { name: 'test user'},
  ]).run(pool);
  expect(user.name).toBe("test user");
})

Reply:
Where in the directory should I create a `db.test.js` file?